protocol/communities_messenger_test.go

```diff
@@ -2172,7 +2172,7 @@ func (s *MessengerCommunitiesSuite) TestLeaveAndRejoinCommunity() {
 			numberInactiveChats++
 		}
 	}
-	s.Require().Equal(3, numberInactiveChats)
+	s.Require().Equal(4, numberInactiveChats)
```

Comment:
Since we now do a complete backup after leaving a community.
Checking id it's possible to do a `community` only backup.